packages/grafana-ui/src/components/Graph/Graph.story.tsx

```diff
@@ -117,7 +117,7 @@ export const withTooltip = () => {
   );
 };
 
-const CustomGraphTooltip: React.FC<TooltipContentProps> = ({ activeDimensions }) => {
+const CustomGraphTooltip = ({ activeDimensions }: TooltipContentProps) => {
```

Comment:
why is this better? using React.FC get more type checking (return type, compatible with React.ComponentType etc)

Reply:
It's not at all. This rule (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md) will be disabled anyways as it's typescript that checks the prop types. It's only in storybook that this needs to be fixed as storybook has this rule enabled (ref https://github.com/grafana/grafana/pull/29053/files#diff-e7e195d3d8fa5b82fa51ed952f3a179d429cbc46c0432ce1f7b357c54c162822R6)

Reply:
We we can also disable this rule for storybook, and I think that's what I would prefer

Reply:
ah, now I see

Reply:
Agree that we'd disable this for storybook as well.